src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -1333,6 +1338,35 @@ public static function createNote($item)
 		return $data;
 	}
 
+	/**
+	 * Fetches the language from the post, the user or the system.
+	 *
+	 * @param array $item
+	 *
+	 * @return string language string
+	 */
+	private static function getLanguage($item)
+	{
+		// Try to fetch the language from the post itself
+		if (!empty($item['language'])) {
+			$languages = array_keys(json_decode($item['language'], true));
```

Comment:
is it possible that `$item['language']` isn't an array and the `array_keys()` method would fail?

Reply:
`$item['language']` should always be a JSON string.

Reply:
is or should ;-) I'd prefer a check, but if you say it isn't necessary, it's fine too

Reply:
I had a look at my data. It seems as if the value there is either `NULL` or some JSON string.